apps/transport/client/stylesheets/components/_map.scss

```diff
@@ -135,16 +135,18 @@
     }
   }
 
-  .general .tile {
-    justify-content: center
+  .general {
```

Comment:
is it normal to have changes in the map CSS?

Reply:
it seems to be the css file for the stats (indeed the name could be more accurate)

Reply:
would you mind renaming it, while we’re at it? (only renaming works, there is an include `**/*` somewhere)

Reply:
done

Reply:
:+1: